docs/submit.md

```diff
@@ -279,3 +280,95 @@ TBD: add docs for such properties here.
+	</array>
+	
+me using a specifically generated provisioning profile...
+
+	<key>DeveloperCertificates</key>
+	<array>
+		<data>MIIFuzCCBKOgAwIBAgIIGUbgCb5Lq+8wDQYJKoZIhvcNAQELBQAwgZYxCzAJBgNVBAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3JsZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3aWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkwHhcNMTkwOTI5MTkzMzIzWhcNMjAwOTI4MTkzMzIzWjCBmTEaMBgGCgmSJomT8ixkAQEMCkwzN1M0WjZCRTkxPDA6BgNVBAMMM0FwcGxlIERpc3RyaWJ1dGlvbjogTmVpbCBLYW5kYWxnYW9ua2FyIChMMzdTNFo2QkU5KTETMBEGA1UECwwKTDM3UzRaNkJFOTEbMBkGA1UECgwSTmVpbCBLYW5kYWxnYW9ua2FyMQswCQYDVQQGEwJVUzCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAOF8e0GnMnPaJCRzg9c6MOjm99ikbHScCIJBcoyXWkH3PkiRrDy106/gjeLDOtG6psbjKl++TGek+7pP85RKwbHn9vCfu3pqcHe+HZ/LNL2puft4VMObuTHvROkpOSMpoxLZd6iyy+9hADSvhZ+8jaAXe+t3Fz3xicbb+IfA729B32Ly6INDsAfrXHXWPM6uVnpvBd66QGuMFUom+u3b0z7+6Ze5zrkqwsDD0lbokACXyP/OWUoAfpzX5ji8QxxEWfzy0UO5VuNysg7mL/ijK+miDo5XzNglD/xSCo2smF7M7OlO4grMv867jmB9H9r60HRneaWqW/m1gd1nXdJT9G0CAwEAAaOCAgYwggICMAwGA1UdEwEB/wQCMAAwHwYDVR0jBBgwFoAUiCcXCam2GGCL7Ou69kdZxVJUo7cwPwYIKwYBBQUHAQEEMzAxMC8GCCsGAQUFBzABhiNodHRwOi8vb2NzcC5hcHBsZS5jb20vb2NzcDAzLXd3ZHIyMDCCAR0GA1UdIASCARQwggEQMIIBDAYJKoZIhvdjZAUBMIH+MIHDBggrBgEFBQcCAjCBtgyBs1JlbGlhbmNlIG9uIHRoaXMgY2VydGlmaWNhdGUgYnkgYW55IHBhcnR5IGFzc3VtZXMgYWNjZXB0YW5jZSBvZiB0aGUgdGhlbiBhcHBsaWNhYmxlIHN0YW5kYXJkIHRlcm1zIGFuZCBjb25kaXRpb25zIG9mIHVzZSwgY2VydGlmaWNhdGUgcG9saWN5IGFuZCBjZXJ0aWZpY2F0aW9uIHByYWN0aWNlIHN0YXRlbWVudHMuMDYGCCsGAQUFBwIBFipodHRwOi8vd3d3LmFwcGxlLmNvbS9jZXJ0aWZpY2F0ZWF1dGhvcml0eS8wFgYDVR0lAQH/BAwwCgYIKwYBBQUHAwMwHQYDVR0OBBYEFID3YsUR7Hv85KrH+OwrG+tCfjrGMA4GA1UdDwEB/wQEAwIHgDATBgoqhkiG92NkBgEHAQH/BAIFADATBgoqhkiG92NkBgEEAQH/BAIFADANBgkqhkiG9w0BAQsFAAOCAQEAu9o1ms4/A9k5uqhbZRHQoqWXY3n+IbbNKKwUe20mhOvw1zdEGG4YhDmcMR0Mz+e+uRPBpc59SXVZMjcD9tiBQsJS4KHXBpaZVx5B7ERSJQNhIH43hvZnZFddFQyN4hA/9c0TTC92ejIdoGDafEA79fYaYPedGxd6Fkp+LEa9uZvnhPN84xGOO1TV+ozKq1Nz1LJaxYLxu57Ux+fNdL/RJMWw0ZB5wXDdYZNW1qMeIZoNXt/Nz6pA9qhiS0dUs46B2mxms1YdQLsGhqE6PZQEoRJChxZRczoK3y5KqmWkElvQFWVM34bNlmAPOZ9I7SaIPWA6EKToyTI9EsRBKAlYKA==</data>
+	</array>
+	
+?? is this what is selected for other apps??
+
+The benefit of using a wildcard cert is that we wouldn't have to worry about creating a cert for each
+individual app
+On the other hand, what is the normal practice for Square??
+...We could check in the actual Square app that we have
+Actual Square Apps use different entitlements for app, watch-appex, watch-app.
+And they use fully qualified application identifiers, keychain groups, etc.
+(Square also leaves rando entitlement template files lying around in their app lol)
```

Comment:
it was a mistake! We cleaned it up eventually ;)

Reply:
For what it's worth, isign has been adding a useless `Entitlements.plist` to every app for 4 years, until this very change.

I'm sorry for the comment, it wasn't very professional. Once I squash and delete this branch it will vanish from history. 

By the way, at Sauce Labs, I would frequently see apps that had similar build scaffolding shipping, right in the app, in one case ballooning its size up to 10%. From very large and prominent companies too.